src/main/kotlin/no/nav/syfo/application/VedtakService.kt

```diff
@@ -57,7 +57,8 @@ class VedtakService(
         val unpublished = vedtakRepository.getUnpublishedInfotrygd()
         return unpublished.map { vedtak ->
             runCatching {
-                infotrygdService.sendMessageToInfotrygd(vedtak)
+                val correlationId = vedtakRepository.getVedtakId(vedtak.uuid)
```

Comment:
Litt kjedelig å måtte bruke en intern database-id som `correlationId` her, men infotrygd liker ikke uuid?

Reply:
Ja, uuid'en blir for lang, i hvert fall hvis man bruker den som String. Kan gjøre et forsøk med et binærformat for uuid'en.

Reply:
Fått testet tilpasningene nå - det fungerer med uuid.